### PR TITLE
Generate integration test certificates on demand

### PR DESCRIPTION
**Pull request by Xtansia**
### Description
Remove the checked in integration test certificates and instead generate them on demand.
This resolves some spurious security alerts caused by having private keys checked in to git.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .ci/generate-certs.sh [hunk 1]
```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
```
Comment (VachaShah) [inline .ci/generate-certs.sh:1]: What about https://github.com/opensearch-project/opensearch-rs/blob/833c177667ca097c7037f6991bbacfa3ee30b757/.ci/certs/generate-certs.sh?

Comment (Xtansia) [reply .ci/generate-certs.sh:1]: This PR deletes the scripts out of `.ci/certs` placing the new one directly in `.ci`
Screenshot 2023-03-16 at 1 15 32 PM

Comment (VachaShah) [reply .ci/generate-certs.sh:1]: Ah! 🤦🏼‍♀️

**Review by VachaShah — APPROVED** [review]
This is awesome!

**Pull request merged**